### PR TITLE
[UTOPIA-905] Table refactor to support two different views - standard and row based

### PR DESCRIPTION
**Pull request by kushal-arora-fw**
# Description

This PR includes the following proposed change(s):

- Table refactor - support standard and row formats now for tabular and row type data
- Updated All tabs - Collection, Use & Disclosure, Storing PI, and Additional Risks

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Validate tabs
image

image

image

image

image

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/sass/_table_row.scss
Comment (Bottle7) [inline src/frontend/src/sass/_table_row.scss:1]: Not a problem, necessarily, but why is this in it's own file separate from the table scss?

File: src/frontend/src/components/common/Table/views/useTableRowView.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { faPlus, faTrash } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import InputText from '../../InputText/InputText';
+import { TableViewProps } from './table-view-props.interface';
+
+type UseTableRowViewProps = TableViewProps;
+
+export const UseTableRowView = (props: UseTableRowViewProps) => {
+  return (
+    <>
```
Comment (Bottle7) [inline src/frontend/src/components/common/Table/views/useTableRowView.tsx:10]: If we already have a parent `<div>` in this component, do we really need the `<>` fragment?

File: src/frontend/src/components/common/Header/Header.tsx [hunk 2]
```diff
@@ -137,7 +137,7 @@ function Header({ user }: Props) {
       <div data-cy="login" className="other">
         {!isAuthenticated() && (
           <button
-            className="bcgovbtn bcgovbtn__secondary--dark"
+            className="bcgovbtn bcgovbtn__secondary--dark extra-gap"
```
Comment (Bottle7) [inline src/frontend/src/components/common/Header/Header.tsx:140]: Where is `.extra-gap` defined? I don't see that class anywhere, but I do see a `.min-gap` defined below. Did you mean to use that?

Comment (kushal-arora-fw) [reply src/frontend/src/components/common/Header/Header.tsx:140]: Ohh no. I intend to remove it only. The default is extra gap now. Thanks for catching!

File: src/frontend/src/sass/_table_row.scss
Comment (kushal-arora-fw) [reply src/frontend/src/sass/_table_row.scss:1]: What do you suggest, how about this?

image

**Review by Bottle7 — APPROVED** [review]

**Pull request merged**